### PR TITLE
Doc fix

### LINKED ISSUE #2100: Setup.py does not conform to newest pip in python requires
In docs
https://readthedocs.com/projects/institute-for-disease-modeling-idmtools/builds/1580520/
```
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in idmtools setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.6.*'
      [end of output]
```

### PR DESCRIPTION
**Pull request by devclinton**
#2100
#2101

### CODE REVIEW

File: idmtools_platform_comps/tests/test_workorder.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def test_wrapper_script_execute_comps(self):
         Returns:
 
         """
-        cmd = "python3.6 --version"
+        cmd = "python3.7 --version"
```
Comment (shchen-idmod) [inline idmtools_platform_comps/tests/test_workorder.py:133]: This needs to be 3.6 for comps test

**Review by shchen-idmod — CHANGES_REQUESTED** [review]

File: idmtools_platform_comps/tests/test_workorder.py [hunk 2]
```diff
@@ -328,7 +328,7 @@ def test_schedule_config_with_wrapper_script_execute_comps(self):
         experiment = Experiment.from_task(wrapper_task, name=self.case_name)
 
         # upload dynamic WorkOrder.json to simulation root dir
-        add_schedule_config(experiment, command="python3.6 --version", node_group_name='idm_abcd', num_cores=1,
+        add_schedule_config(experiment, command="python3.7 --version", node_group_name='idm_abcd', num_cores=1,
```
Comment (shchen-idmod) [inline idmtools_platform_comps/tests/test_workorder.py:331]: This needs to be 3.6 for comps test

File: idmtools_platform_comps/tests/test_workorder.py [hunk 3]
```diff
@@ -315,7 +315,7 @@ def test_schedule_config_with_wrapper_script_execute_comps(self):
         Returns:
 
         """
-        cmd = "python3.6 --version"
+        cmd = "python3.7 --version"
```
Comment (shchen-idmod) [inline idmtools_platform_comps/tests/test_workorder.py:318]: This needs to be 3.6 for comps test

File: idmtools_platform_comps/tests/test_workorder.py [hunk 4]
```diff
@@ -154,7 +154,7 @@ def test_wrapper_script_execute_comps(self):
                 if asset.filename in ["stdout.txt"]:
                     content = asset.content.decode('utf-8').replace("\\\\", "\\")
                     # don't check full version in case comps updates system
-                    self.assertIn('Python 3.6', content)
+                    self.assertIn('Python 3.7', content)
```
Comment (shchen-idmod) [inline idmtools_platform_comps/tests/test_workorder.py:157]: This needs to be 3.6 for comps test

File: idmtools_platform_comps/tests/test_workorder.py [hunk 5]
```diff
@@ -340,7 +340,7 @@ def test_schedule_config_with_wrapper_script_execute_comps(self):
...
```
Comment (shchen-idmod) [inline idmtools_platform_comps/tests/test_workorder.py:343]: This needs to be 3.6 for comps test

**Review by shchen-idmod — CHANGES_REQUESTED** [review]

File: idmtools_platform_comps/tests/test_workorder.py (hunk 3 above)
Comment (devclinton) [reply idmtools_platform_comps/tests/test_workorder.py:318]: Fixed

File: idmtools_platform_comps/tests/test_workorder.py (hunk 2 above)
Comment (devclinton) [reply idmtools_platform_comps/tests/test_workorder.py:331]: Fixed

File: idmtools_platform_comps/tests/test_workorder.py (hunk 4 above)
Comment (devclinton) [reply idmtools_platform_comps/tests/test_workorder.py]: Fixed @shchen-idmod

File: idmtools_platform_comps/tests/test_workorder.py (hunk 5 above)
Comment (devclinton) [reply idmtools_platform_comps/tests/test_workorder.py]: Fixed @shchen-idmod

**Comment by shchen-idmod** [comment]
All tests passed in Jenkins

**Review by shchen-idmod — APPROVED** [review]

**Review by ZDu-IDM — APPROVED** [review]

**Pull request merged**